library/rethinkdb

```diff
@@ -1,31 +1,32 @@
+1.15: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.15.3
+1.16.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.0
+1.16.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.1
+1.16.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.2
+1.16.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.3
+1.16: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.3
+1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/1.16.3
+2.0.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.0
+2.0.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.1
+2.0.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.2
+2.0.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.3
+2.0.4: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.4
+2.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.0.4
+2.1.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.0
+2.1.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.1
+2.1.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.2
+2.1.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.3
+2.1.4: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.4
+2.1.5: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.5
+2.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.1.5
+2.2.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.0
+2.2.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+2.2.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
+2.2.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
+2.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+2: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.1
+latest: git://github.com/rethinkdb/rethinkdb-dockerfiles@9244272dd4cdaea7b61b2321f827b2a70c4cfa9f jessie/2.2.2
```

Comment:
Hmm, we missed this last time too (:disappointed: in myself).  These last 4 lines should have the folder at the end bumped to `2.2.3`.